pkg/trace/api/api.go

```diff
@@ -518,6 +546,8 @@ type Payload struct {
 	// ClientComputedStats reports whether the client has computed and sent over stats
 	// so that the agent doesn't have to.
 	ClientComputedStats bool
+	// ClientDroppedP0s reports the number of P0 traces chunks dropped by the client
```

Comment:
```suggestion
	// ClientDroppedP0s specifies the number of P0 traces chunks dropped by the client.
```
We usually used "reports" for `bool`.